Super Tafani Member/index.html

```diff
@@ -0,0 +1,220 @@
+               <p class="grey mb-30">Manages tafani members and internacl customer activities and hide/show posts</p>
+               </div>
+             <div class="d-flex  gap-3 ">
+                <div class="input-group w-35 border-grayish-blue   mb-3" >
+                    <span class="input-group-text pt-3    border-0 lightgrey-bg" id="basic-addon1">
+                        <svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
+                            <path d="M21.7099 20.29L17.9999 16.61C19.44 14.8144 20.1374 12.5353 19.9487 10.2413C19.76 7.94733 18.6996 5.81281 16.9854 4.27667C15.2713 2.74053 13.0337 1.91954 10.7328 1.9825C8.43194 2.04546 6.24263 2.98759 4.61505 4.61517C2.98747 6.24275 2.04534 8.43207 1.98237 10.7329C1.91941 13.0338 2.74041 15.2714 4.27655 16.9855C5.81269 18.6997 7.94721 19.7601 10.2412 19.9488C12.5352 20.1375 14.8143 19.4401 16.6099 18L20.2899 21.68C20.3829 21.7738 20.4935 21.8482 20.6153 21.8989C20.7372 21.9497 20.8679 21.9758 20.9999 21.9758C21.1319 21.9758 21.2626 21.9497 21.3845 21.8989C21.5063 21.8482 21.6169 21.7738 21.7099 21.68C21.8901 21.4936 21.9909 21.2444 21.9909 20.985C21.9909 20.7257 21.8901 20.4765 21.7099 20.29ZM10.9999 18C9.61544 18 8.26206 17.5895 7.11091 16.8203C5.95977 16.0511 5.06256 14.9579 4.53275 13.6788C4.00293 12.3997 3.86431 10.9923 4.13441 9.63439C4.4045 8.27653 5.07119 7.02925 6.05016 6.05028C7.02912 5.07131 8.27641 4.40463 9.63427 4.13453C10.9921 3.86443 12.3996 4.00306 13.6787 4.53287C14.9578 5.06268 16.051 5.95989 16.8202 7.11103C17.5894 8.26218 17.9999 9.61556 17.9999 11C17.9999 12.8565 17.2624 14.637 15.9497 15.9498C14.6369 17.2625 12.8564 18 10.9999 18Z" fill="#8E9AA0"/>
+                            </svg>
+                    </span>
+                    <input type="text" class="form-control ms-0 pt-3 pb-2 lightgrey-bg border-0" placeholder="Search By keyword" aria-label="Username" aria-describedby="basic-addon1">
+                  </div>
+             <div class=" w-28">
+                <select class="form-select border-0 py-3 lightgrey-bg" aria-label="Disabled select example" disabled>
+                    <option selected>Status</option>
+                    <option value="1">One</option>
+                    <option value="2">Two</option>
+                    <option value="3">Three</option>
+                  </select>
+             </div>
+                <div class="w-28">
+                    <select class="form-select py-3 border-0 " aria-label="Disabled select example" disabled>
+                        <option selected>Date</option>
+                        <option value="1">One</option>
+                        <option value="2">Two</option>
+                        <option value="3">Three</option>
+                      </select>
+                </div>
+                <div class="w-28 mt-2">
+                    <div class="pill" role="group" aria-label="Basic mixed styles example">
```

Comment:
Please check the space between this `pill `and the `select` element; it needs to be **increased** slightly

Reply:
done